csci-499-sp24/FaceStylizer
# Creating FaceStylizer Gallery Component

Created standalone gallery component with timestamp and gallery header

Testing Instructions:

1. cd into client
2. npm run dev 
3. if you add the gallery component directly below the file upload component in app.js you'll see the gallery component on homepage for testing purposes 

`function App() {`
    `return (`
       ` <div>`
            `<RHeader/>`
            `<CenteredImage/>`
           ` <FileUpload/>`
          `  <Gallery/>`
      `  </div>`
  `  );`
`}`

5. users should see gallery title, gallery subheading, gallery table component and timestamps
